docs/reference/cvat/custom-models.md

```diff
@@ -51,11 +51,16 @@ We will walk through updating the [DEtection TRansformer (DETR)](https://github.
 
 In this step, you will launch a JupyterLab Workspace in Onepanel to test and adjust your code before it is added to the the CVAT training Workflow Template. The JupyterLab Workspace Template just like the CVAT training Workflow Template, uses the `onepanel/dl` Docker image which has both PyTorch 1.6 and TensorFlow 2.3 installed and provides a consistent environment for testing and deploying your training code.
 
-1. Fork the [DEtection TRansformer (DETR)](https://github.com/facebookresearch/detr) repository.
+1. From **CVAT** assuming you have the data and annotations set up, export data & annotations by clicking  
+**Actions** > **Export as dataset** > **MS COCO**.
 
-2. Launch a **JupyterLab** Workspace on a GPU node pool, then [clone](/docs/reference/jupyterlab/git#cloning) your fork. You can optionally run on a CPU node pool but it will take much longer to test.
+2. You can then sync the output into your object storage to use for training.
```

Comment:
@inohmonton99 You probably want to add more detail here around copying the path and then putting that path in **Workspace path** and then clicking **Sync to object storage**